pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -287,9 +286,7 @@ def ddp_train(self, process_idx, model):
     def configure_ddp(
         self, model: LightningModule, device_ids: List[int]
     ) -> DistributedDataParallel:
-        model = LightningDistributedDataParallel(
-            model, device_ids=device_ids, find_unused_parameters=True
-        )
+        model = self.ddp_plugin.configure_ddp(model, device_ids)
```

Comment:
do you need to check that `self.ddp_plugin` is not `None` ? it'd be nice to have the typehint for it as `Optional[DDPPlugin]`

Reply:
the root accelerator ensures this should not be none.

but tests are failing for some reason where one is none.... trying to figure out why

Reply:
found it... pickle state snuck in there haha